scss/_layouts_application-panels.scss

```diff
@@ -0,0 +1,36 @@
+@mixin vf-l-application-panels {
+  $header-padding-bottom: $sp-unit * 6;
+
+  .p-panel,
+  .p-panel--aside {
+    background: $colors--light-theme--background-default;
+    color: $colors--light-theme--text-default;
+    min-height: 100%; // make sure it fills whole container even when has little content
+
+    &.is-dark {
+      background: $colors--dark-theme--background-default;
+      color: $colors--dark-theme--text-default;
+    }
+  }
+
+  .p-panel__header,
+  .p-panel__header--sticky {
+    padding-bottom: $header-padding-bottom;
+    padding-top: $spv-outer--medium;
+  }
+
+  .p-panel__header--sticky {
+    background: linear-gradient(to bottom, #{$colors--light-theme--background-default} 0%, #{$colors--light-theme--background-default} 70%, transparent 100%);
+    position: sticky;
+    top: 0;
+    z-index: 1;
+
+    .is-dark & {
+      background: linear-gradient(to bottom, #{$colors--dark-theme--background-default} 0%, #{$colors--dark-theme--background-default} 70%, transparent 100%);
+    }
+  }
+
+  .p-panel__content {
+    // TODO: not needed for now?
```

Comment:
Can we remove for now please

Reply:
Ah sure, forgot about this. I wonder if SCSS renders it or is it smart enough to skip it...

Reply:
SCSS is smart enough to ignore it but makes the codebase tidy.